### PR TITLE
add a file for owatonna

### PR DESCRIPTION
**Pull request by mike-hines**
The purpose of this PR is to add a hometown file for Owatonna, MN

Some :question: that I want you all to help me answer:

1.  What kind of food do you like?
2.  If you've been to Owatonna what did you do?

### CODE REVIEW

File: mike-hines-owatonna.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Owatonna, MN
+
+### Restaurants
```
Comment (timothy-grimes) [inline mike-hines-owatonna.md:3]: this is just beautiful

**Review by timothy-grimes — APPROVED** [review]
1. pizza
2. I have not.

**Comment by mike-hines** [comment]
The pull request resolves #13

**Comment by timothy-grimes** [comment]
I already reviewed and approved so you're good to go @mike-hines

**Comment by timothy-grimes** [comment]
:shipit: :shipit: :shipit: :shipit: :shipit: :shipit:

**Pull request merged**